api/http/workflow_test.go

```diff
@@ -105,6 +140,180 @@ func TestWorkflowFoundAndLaunchedWithParameters(t *testing.T) {
 	req, _ := http.NewRequest("POST", "/api/v1/workflow/test", strings.NewReader(payload))
 	router.ServeHTTP(w, req)
 
-	assert.Equal(t, 201, w.Code)
+	assert.Equal(t, http.StatusCreated, w.Code)
 	assert.Equal(t, 1, len(dataStore.Jobs))
+
+	var response map[string]string
+	body := w.Body.Bytes()
+	err = json.Unmarshal(body, &response)
+	value, ok := response["id"]
+	assert.True(t, ok)
+
+	w = httptest.NewRecorder()
+	req, _ = http.NewRequest("GET", fmt.Sprintf("/api/v1/workflow/test/%s", value), nil)
```

Comment:
we are including fmt only for this one call, cant we just concatenate the strings here?

Reply:
@merlin-northern right, I've removed the usage of `fmt` and improved the test code using the constant used to define the URLs in the router.